openqaoa/workflows/managed_jobs.py

```diff
@@ -0,0 +1,149 @@
+#   Copyright 2022 Entropica Labs
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import os
+import json
+# from braket.jobs import save_job_result
+
+from openqaoa.devices import DeviceAWS
+from openqaoa.workflows.optimizer import Optimizer, QAOA, RQAOA
+from openqaoa.problems.problem import QUBO
+
+class Aws_job(Optimizer):
```

Comment:
Since its a class name, shall we can go with `AWSJobs` or AwsJobs`?

Reply:
I think you have a good point: it should probably be `AWSJobs`. I will change it during the next iteration

Reply:
Done it!